coalib/testing/LocalBearTestHelper.py

```diff
@@ -185,10 +201,11 @@ def check_results(self,
                                   create_tempfile=create_tempfile,
                                   tempfile_kwargs=tempfile_kwargs,
                                   settings=settings)
+        self.addTypeEqualityFunc(list, self.assertObjectsEqual)
```

Comment:
1. This should be run somewhere in the beginning of a base test class or so, otherwise it takes effect only in `check_validity`
2. `list` doesn't use `generate_eq` or `generate_repr`, so this is not really working